yb-voyager/cmd/analyzeSchema.go

```diff
@@ -169,10 +170,14 @@ func reportBasedOnComment(comment int, fpath string, issue string, suggestion st
 
 // adding migration summary info to reportStruct from summaryMap
 func reportSummary() {
+
+	//reading source db metainfo
+	sourceDBMetaInfo := openMetaInfoJsonFile()
```

Comment:
As suggested in other comments, when you rename the type to `MigInfo`, the object should be called as just `miginfo` and not `sourceDBMigInfo`.
Going forward, MigInfo may contain information other than source db.